.github/workflows/unittests.yml

```diff
@@ -77,34 +27,21 @@ jobs:
         run: larq_compute_engine/tflite/build_make/build_lce.sh --native --pip
       - name: "Install TF Lite Python Runtime Package"
         run: pip3 install /tmp/tflite_pip/python3/dist/*.whl
-      - name: "Python Test Dependency: Build Bazel Package Creator Binary"
-        run: ./bazelisk build :build_pip_pkg
-      - name: "Python Test Dependency: Build LCE PIP Package"
-        run: bazel-bin/build_pip_pkg artifacts
-      - name: "Python Test Dependency: Install LCE PIP Package"
-        run: pip3 install larq pytest pytest-xdist artifacts/*.whl
-      - name: "Run Python Tests"
-        run: |
-          # Go to a different directory to force it to use the installed package
-          # and not use the local larq_compute_engine directory
-          cd larq_compute_engine/tflite
-          python3 -m pytest . -n auto
 
   ARM:
     runs-on: ubuntu-latest
-    container:
-      image: tensorflow/tensorflow:custom-op-ubuntu16
-
```

Comment:
This should speedup CI

Reply:
Note that the `configure.sh` script, in case of `manylinux2010`, adds some `crosstool_top` flag to `.bazelrc` that refers to a hardcoded path in that docker container. However, since this CI run only uses `--config=rpi3` and `--config=aarch64`, those flags are overwritten.
I'm indeed in favor of removing the container here, but maybe we should add a comment that this means we can only use bazel commands with one of those configs and not the default one!

Reply:
I am not sure if I follow what you mean exactly, do you want to add this comment in a followup PR?

Reply:
I meant that either in this PR or a future PR, it might be nice to have a comment in this `unittests.yml` file, with something like:
```
# Note that the default bazel config will not work here because this does not run in the docker container. Only the `rpi3` and `aarch64` config options are valid here.
```

Reply:
Just for reference, you can build it outside the container without the manylinux2010 toolchaing with no problem: https://github.com/larq/compute-engine/actions/runs/38647228/workflow#L67

Reply:
Ah yes thats true. The hardcoded toolchain path only gets added for manylinux2010.